core-bundle/src/Security/ContaoCorePermissions.php

```diff
@@ -186,4 +186,39 @@ final class ContaoCorePermissions
      * Subject must be a corresponding group ID or an array of group IDs.
      */
     public const MEMBER_IN_GROUPS = 'contao_member.groups';
+
+    /**
+     * Prefix for all DCA related permission attributes.
+     */
+    public const DCA_PREFIX = 'contao.dca.';
```

Comment:
```suggestion
    public const DCA_PREFIX = 'contao_dca.';
```
Would this be more consistent with the other prefixes `contao_member` and `contao_user`?